static/js/activity.js

```diff
@@ -200,6 +205,13 @@ function focus_lost() {
     // server; instead, we wait for our next periodic update, since
     // this data is fundamentally not timely.
     exports.has_focus = false;
+
+    // If the webapp is determined to be idle, we check if the desktop app
+    // is in focus in some other server, and request the desktop app to update
+    // desktop_has_focus variable.
+    if (window.electron_bridge !== undefined) {
+        window.electron_bridge.send_event('electron_presence_update');
```

Comment:
Hmm, why do you want an event when focus is lost?

Reply:
When according the webapp the user is idle, we'd want the desktop app to check whether the user is actually idle or using some other organization (has the desktop app opened)  and hence we send a request to the desktop app to this via this event.